lti/models/application_instance.py

```diff
@@ -26,9 +26,12 @@ def __repr__(self):
 def build_shared_secret():
     return secrets.token_hex(64)
 
+def build_unique_key():
+  return lti_key + secrets.token_hex(16)
```

Comment:
In our other apps this is usually a human readable string something like `lbs-atomicsearch`. I wonder if there is a reason why we make them human readable or if that is just a convention we came up with.

Reply:
It's a convention we came up with. The key can be anything you like. We decided to make it something that would be easy for a user to read, but it can be an auto generated value.

Reply:
Awesome!